Scripts/RemediationScripts/AutoRemediation.ps1

```diff
@@ -161,8 +161,8 @@ Enter the choice (1|2)";
                 ($control.ControlId -eq "Azure_ServiceBus_DP_Use_Secure_TLS_Version") -or
                 ($control.ControlId -eq "Azure_DBForMySQLFlexibleServer_DP_Use_Secure_TLS_Version_Trial") -or
```

Comment:
Have you updated the details for these controls in control specification JSON file and the readme file?

Reply:
No Saurabh, Its showing because of spacing only.